.rubocop.yml

```diff
@@ -4,6 +4,7 @@ require:
   - standard
   - rubocop-rails
   - rubocop-rspec
+  - rubocop-capybara
```

Comment:
what notice is being quieted w/ this change?

Reply:
```
The following RuboCop extension libraries are installed but not loaded in config:
  * rubocop-capybara

You can opt out of this message by adding the following to your config (see https://docs.rubocop.org/rubocop/extensions.html#extension-suggestions for more options):
  AllCops:
    SuggestExtensions: false
```